docs/docsite/rst/contributor_guide.rst

```diff
@@ -30,23 +30,25 @@ Contributing documentation
 
 Additions to the collection documentation are very welcome! We have three primary types of documentation, each with their own syntax and rules.
 
-Module and plugin documentation
+README and other markdown files
 -------------------------------
 
-This type of documentation gets generated from structured YAML, inside of a Python string. It is included in the same code that it's documenting, or in a separate Python file, such as a doc fragment. Please see the :ref:`module format and documentation guidance <developing_modules_documenting>` for more information.
+Markdown files (those with the extension ``.md``) can be found in several directories within the repository. These files are primarily aimed at developers and those browsing the repository, to explain or give context to the other files nearby.
+
+The main exception to the above is the ``README.md`` in the repository root. This file is more important because it provides introductory information and links for anyone browsing the repository, both on GitHub and on the collection's `Ansible Galaxy page <https://galaxy.ansible.com/community/hashi_vault>`_.
 
-This type of documentation is tested with ``ansible-test sanity`` and full instructions are available on the :ref:`testing module documentation <testing_module_documentation>` page.
+Markdown files can be previewed natively in GutHub, so they are easy to validate by reviewers, and there are no specific tests that need to run against them.
```

Comment:
typo "GutHub".

Reply:
🤦‍♂️I have been doing this a LOT lately, thank you.